joomla/joomla-cms
# [4.4] Remove unused cassiopeia inline event handler

### Summary of Changes
The current Cassiopeia version has an inline onload event handler for the fontscheme.current css file. The inline event handler produces a warning in strict CSP mode and is redundant, as the lazy-loading functionality (that was the purpose of the handler) is handled in the template.js and trigger by the rel="lazy-stylesheet" attribute.


### Testing Instructions
* Configure the "roboto (local)" font as font in cassiopeia, verify that it's loaded by the browser using the browser dev tools.
* Configure a strict ("script-src: 'self' 'unsafe-eval'") report-only script CSP using the Joomla HTTP Header plugin:

<img width="1425" alt="Bildschirmfoto 2024-06-01 um 14 20 13" src="https://github.com/joomla/joomla-cms/assets/498096/42058d43-500e-4791-b7cf-36e889980c13">




### Actual result BEFORE applying this Pull Request
* CSS file loaded
* Error message about forbidden unsafe-inline JS calls, caused by an "onload" handler of the mentioned CSS file in the browser console shown

<img width="1512" alt="Bildschirmfoto 2024-06-01 um 14 21 00" src="https://github.com/joomla/joomla-cms/assets/498096/95be35e4-0578-46cb-9eae-81f3dd4fae76">


### Expected result AFTER applying this Pull Request
* CSS file loaded
* Error message about unsafe-inline calls about the onload handler is gone


### Link to documentations
Please select:
- [x ] No documentation changes for docs.joomla.org needed
